tutorial/errors.po

```diff
@@ -29,6 +30,8 @@ msgid ""
 "tried out the examples you have probably seen some.  There are (at least) "
 "two distinguishable kinds of errors: *syntax errors* and *exceptions*."
 msgstr ""
+"到目前為止還沒有提到錯誤訊息，但如果你嘗試這些範例，你可能會發現一些錯誤訊"
+"息。常見有（至少）兩種明顯的錯誤是 *語法錯誤* 和 *例外*。"
```

Comment:
The spaces here should be written as <code>兩種明顯的錯誤是\\ \*語法錯誤\*\\ 和\\ \*例外\*\\ 。</code>, so that there will be no redundant space between characters in the generated output.  Also, could you please quote the original English for "syntax error" and "exception" after each? Like: `語法錯誤（syntax error）`, btw sorry for not mentioning this in the previous review.